src/components/admonition/admonition.scss

```diff
@@ -0,0 +1,41 @@
+.admonition {
+    display: flex;
+    width: 100%;
```

Comment:
`display:flex` already makes this element block-level, so setting `width:100%` shouldn't be necessary, and can lead to problems. Was there something that didn't work as it should without this?

https://www.youtube.com/watch?v=-st14lUQD3U

Reply:
I'll have to do this during the cooldown, but have you checked how we do it in Kompendium? I don't want to re-invent the wheel 😅 https://github.com/jgroth/kompendium/blob/master/src/components/markdown/markdown.scss

Reply:
Those are just admonitions inside markdown though, while this is a separate "admonition component". Although, one way to implement it would certainly be to stick the input in some markdown, which is then rendered with the markdown admonition plugin. One significant benefit with that would also be that if we want to support admonitions in markdown elsewhere, like in the history feed, we can use the same plugin, and thus get identical admonitions.

Reply:
That's an implementation issue though, so we don't _have_ to change that now. However, I think we might want to check what types are supported there, and limit this component to those types in step 1. That way we can postpone the decision on whether to use markdown admonitions, without having to make a breaking change if we decide to go that route.